element_animal/surgery.py

```diff
@@ -0,0 +1,169 @@
+                            database if it does not yet exist.
+        create_tables (bool): when True (default), create tables in the
+                            database if they do not yet exist.
+        linking_module (bool): a module name or a module containing the
+        required dependencies to activate the `subject` element:
+
+    Dependencies:
+    Upstream tables:
+        User: the who conducted a particular surgery/implantation
+    """
+
+    if isinstance(linking_module, str):
+        linking_module = importlib.import_module(linking_module)
+    assert inspect.ismodule(
+        linking_module
+    ), "The argument 'linking_module' must be a module's name or a module"
+
+    global _linking_module
+    _linking_module = linking_module
+
+    subject.activate(
+        subject_schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+        linking_module=linking_module,
+    )
+    schema.activate(
+        surgery_schema_name,
+        create_schema=create_schema,
+        create_tables=create_tables,
+        add_objects=linking_module.__dict__,
+    )
+
+
+@schema
+class CoordinateReference(dj.Lookup):
+    """Coordinate reference system
+
+    Attributes:
+        reference ( varchar(60) ): Reference system (e.g., bregma, lambda, etc.)
+    """
+
+    definition = """
+    reference   : varchar(60)
+    """
+    contents = zip(
+        ["bregma", "lambda", "dura", "skull_surface", "sagittal_suture", "sinus"]
+    )
+
+
+@schema
+class BrainRegion(dj.Manual):
+    """Brain region of a given surgery
+
+    Attributes:
+        region_acronym ( varchar(32) ) : Brain region shorthand
+        region_name ( varchar(128) ) : Brain region full name
+    """
+
+    definition = """
+    region_acronym : varchar(32)   # Brain region shorthand
+    ---
+    region_name    : varchar(128)  # Brain region full name
+    """
+
+
+@schema
+class Hemisphere(dj.Lookup):
+    """Brain region hemisphere
+
+    Attributes:
+        hemisphere ( varchar(8) ): Brain region hemisphere (e.g., left, right, middle)
+    """
+
+    definition = """
+    hemisphere: varchar(8) # Brain region hemisphere
+    """
+
+    contents = zip(["left", "right", "middle"])
+
+
+@schema
+class ImplantationType(dj.Lookup):
+    """Type of implantation
+
+    Attributes:
+        implant_type ( varchar(16) ): Short name for type of implanted device
+        implant_description ( varchar(32) ): Full description for implanted device
+    """
+
+    definition = """
+    implant_type        : varchar(16) # Short name for type of implanted device
+    ---
+    implant_description : varchar(32) # Full description for implanted device
+    """
+
+    contents = (
+        ("ephys", "electophysiology"),
+        ("fiber", "fiber photometry"),
+        ("opto", "optogenetic pertubation"),
+    )
+
+
+@schema
+class Implantation(dj.Manual):
+    """Implantation of a device
```

Comment:
@ttngu207 Can we also have a table named Injection somewhere that can support multiple injectibles (e.g., drug, virus, tracer, etc) and types (e.g., i.p, intracranial, subq etc.) so that it can be a bit more task-agnostic?